modules/nw-egressnetworkpolicy-about.adoc

```diff
@@ -37,6 +37,29 @@ ifdef::ovn[]
 - A protocol that is one of the following protocols: TCP, UDP, and SCTP
 endif::ovn[]
 
+[IMPORTANT]
+====
+You must allow access to the {product-title} API servers or connections to the API endpoints are blocked. For example, you must define egress firewall rules similar to the following:
```

Comment:
Looks like I am not fully get this sentence.  I think the context for this issue is: In OVN cluster, there is an egressfirewall in one namespace and there might be a deny rule to 0.0.0.0/0, then pods from that namespace cannot access the API servers. To make it work, have to edit the existing egressfirewall and add allow rule to API endpoints. Like following rules.